### DISCUSSION TITLE
reclassify infectious disease terms

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#3862] [Bigger project]
Per the Mondo developers call on 06/11, we propose to treat parasite as a synonym of eukaryote
- [ ] to do: add parasite logical axiom w/ eukaryote

**Question: ** can anyone think of a eukaryotic infectious disease that is not a parasitic disease?

**Question: ** Should we put fungal infectious disease under parasitic infectious disease?

- [ ] to do: classify all children of infectious disease as a child of bacterial infectious disease, viral infectious disease or parasitic infectious disease?

cc @cmungall @mellybelly @pnrobinson @sabrinatoro

**Comment by pnrobinson** [comment]
Fungi are eukaryotes but are not considered parasites.
Fungi are opportunistic

**Comment by maglott** [comment]
Shouldn't  synonyms be reciprocal?  Not all eukaryotes are parasites.
Amoebae are also eukaryotes, and they are not fungi

what does "classify all children of infectious disease as a child of bacteria, virus or parasite/eukaryote" mean?
Seems odd to classify any disorder as a child of an organism.

**Comment by nicolevasilevsky** [comment]
@maglott I meant: bacterial infectious disease, viral infectious disease or parasitic infectious disease. Yes, not as children of organisms, that would be odd! :)

**Comment by maglott** [comment]
And going back to the original question... if we think of parasites as describes in  https://en.wikipedia.org/wiki/Parasitism, I don't think all fungal infections are categorized as parasitic.  For example, https://www.ncbi.nlm.nih.gov/mesh/68002177  is not treed under parasitic diseases. https://www.ncbi.nlm.nih.gov/mesh/68010272

**Comment by pnrobinson** [comment]
fungal infections are **not** considered to be parasitic infections

**Comment by nicolevasilevsky** [comment]
Thanks for your feedback @pnrobinson and @maglott 

The next open question is:
- how to better define parasitic infection (textually and can it be defined logically?)
Ask experts from https://veupathdb.org/veupathdb/app

**Comment by pnrobinson** [comment]
Isn't there an ontology term for parasite that we can refer to?
The definition could be simply an  disease caused by infection by a parasite.

**Comment by cmungall** [comment]
@pnrobinson 
> fungal infections are not considered to be parasitic infections

is this true for all fungal infections, or just some?

currently many sources treat for example Cryptosporidium infection as parasitic e.g

 - CDC: https://www.cdc.gov/parasites/crypto/index.html
 - ORDO: https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_1549

are these formally wrong?

If it is the case that *some* fungal infections are parasitic infections, what is the differentia?

Our current def is from NCIT:

_A successful invasion of a host by an organism that uses the host for food and shelter_

As @nicolevasilevsky says we need a more precise definition here

**Comment by cmungall** [comment]
> Isn't there an ontology term for parasite that we can refer to?
https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FRO_0002444

But it lacks a definition. Someone contact the Term Editor and bug them.

oh hold on...

![image](https://user-images.githubusercontent.com/50745/124307920-5a45f200-db1d-11eb-8be8-d3a1c79fe07b.png)

**Comment by nicolevasilevsky** [comment]
> oh hold on...


haha

**Comment by cmungall** [comment]
http://purl.obolibrary.org/obo/IDO_0000403

_A symbiont role borne by an organism in virtue of the fact that it derives a growth, survival, or fitness advantage from symbiosis while the other symbiont's growth, survival, or fitness is reduced._

Where symbiosis is _An interaction between two organisms living together in more or less intimate association_

Well it's a bit more precise than the current mondo definition but still hard to apply operationally.

 - derives a growth, survival, or fitness advantage from symbiosis: fits all fungal infections
 - the other symbiont's growth, survival, or fitness is reduced: fits many fungal infections, not just Cryptosporidium
 - living together in more or less intimate association: ?? who knows. I don't know what it means to be intimate with a fungus that is infecting me, and why I would be more intimate with Cryptosporidium than other fungi

**Comment by pnrobinson** [comment]
I am not up to date on this but is it certain that cryptosporidium is a fungus? https://www.cdc.gov/parasites/crypto/index.html

An important difference is that funguses can live without a human host and are opportunistic infectious agents. 

The word parasite can be interpreted as a role or as a characteristic of phyla, e.g. the phylum of cryptosporidium, The Apicomplexa (also called Apicomplexia) are a large phylum of parasitic alveolates. 
https://en.wikipedia.org/wiki/Apicomplexa

I think for mondo we do not want to include the "role" of being a parasite, but rather want to use the biological definitions.

**Comment by nicolevasilevsky** [comment]
cryptosporidium was a mistake, it's a parasitic disease. :)
